test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -27,5 +27,6 @@ public void isSimilar_returnsTrue() throws IllegalValueException
 	{
 		assertTrue(name.isSimilar(new Name("John")));
 		assertTrue(name.isSimilar(new Name("John K SMITh")));
+		assertTrue(name.isSimilar(new Name("Smith, John K")));
```

Comment:
You should put all your test cases in different methods with clear method name. If you do it like this, later you might forgot what you want to test at each lines above.